src/ResourceManager/StorageSync/Commands.StorageSync.Test/Commands.StorageSync.Test.csproj

```diff
@@ -35,6 +35,27 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <None Include="app.config" />
+    <None Include="ScenarioTests\ServerEndpointTests.ps1">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <None Include="ScenarioTests\CloudEndpointTests.ps1">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <None Include="ScenarioTests\RegisteredServerTests.ps1">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <None Include="ScenarioTests\SyncGroupTests.ps1">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <None Include="ScenarioTests\StorageSyncServiceTests.ps1">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <Compile Include="ScenarioTests\ServerEndpointTests.cs" />
```

Comment:
@ankushbindlish2 CI is currently failing because the corresponding `.json` files containing the recordings for these tests were not added to this `.csproj` file and are not being copied to the output directory. It should look like the following for each of the `.json` files:

```
<None Include="SessionRecords\Microsoft.Azure.Commands.StorageSync.Test.ScenarioTests.StorageSyncTests\TestStorageSync.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
    </None>
```